website/config/routes.js

```diff
@@ -180,7 +171,33 @@ module.exports.routes = {
     },
   },
 
+  'GET /try-fleet/register': {
+    action: 'try-fleet/view-register',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    }
+  },
 
+  'GET /try-fleet/login': {
+    action: 'try-fleet/view-sandbox-login',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    }
+  },
+
+  'GET /try-fleet/sandbox': {
+    action: 'try-fleet/view-sandbox-or-redirect',
```

Comment:
`action: 'try-fleet/view-sandbox-teleporter-or-redirect-because-expired'`

(See my review comments in the corresponding page script for more info.)